lib/src/path_observer.dart

```diff
@@ -290,10 +290,10 @@ _getObjectProperty(object, property) {
     // whether the type practically implements the indexer API
     // (smoke.hasInstanceMethod(type, const Symbol('[]')))?
     if (object is Indexable || object is Map && !_MAP_PROPERTIES.contains(property)) {
-      return object[smoke.symbolToName(property)];
+      return object[property];
     }
     try {
-      return smoke.read(object, property);
+      return jsProxyReflectable.reflect(object).invokeGetter(property);
```

Comment:
it seems a bit odd to have a dependency on Polymer. In particular, this would break  usage of package:observe outside of the browser.

is there any reason we are using this type from Polymer, instead of using reflectable directly?

CC @jakemac53